study.md

```diff
@@ -134,15 +134,15 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, give a brief description of what HTTP is.
 
 ```md
-<!-- your answer here -->
+HTTP is the set of rules (protocols) that determine how web pages are displayed and formatted
```

Comment:
More accurately it is how clients send requests, and servers make responses, and how they exchange resources which often is a web page.